app/i18n/ru_RU.utf8/gen.php

```diff
@@ -0,0 +1,181 @@
+		'mark_favorite' => 'Mark as favourite',
+		'remove' => 'Remove',
+		'see_website' => 'See website',
+		'submit' => 'Submit',
+		'truncate' => 'Delete all articles',
+	),
+	'auth' => array(
+		'email' => 'Email address',
+		'keep_logged_in' => 'Keep me logged in <small>(1 month)</small>',
+		'login' => 'Login',
+		'login_persona' => 'Login with Persona',
+		'login_persona_problem' => 'Connection problem with Persona?',
+		'logout' => 'Logout',
+		'password' => array(
+			'_' => 'Password',
+			'format' => '<small>At least 7 characters</small>',
+		),
+		'registration' => array(
+			'_' => 'New account',
+			'ask' => 'Create an account?',
+			'title' => 'Account creation',
+		),
+		'reset' => 'Authentication reset',
+		'username' => array(
+			'_' => 'Username',
+			'admin' => 'Administrator username',
+			'format' => '<small>maximum 16 alphanumeric characters</small>',
+		),
+		'will_reset' => 'Authentication system will be reset: a form will be used instead of Persona.',
+	),
+	'date' => array(
+		'Apr' => '\\A\\p\\r\\i\\l',
+		'Aug' => '\\A\\u\\g\\u\\s\\t',
+		'Dec' => '\\D\\e\\c\\e\\m\\b\\e\\r',
+		'Feb' => '\\F\\e\\b\\r\\u\\a\\r\\y',
+		'Jan' => '\\J\\a\\n\\u\\a\\r\\y',
+		'Jul' => '\\J\\u\\l\\y',
+		'Jun' => '\\J\\u\\n\\e',
+		'Mar' => '\\M\\a\\r\\c\\h',
+		'May' => '\\M\\a\\y',
+		'Nov' => '\\N\\o\\v\\e\\m\\b\\e\\r',
+		'Oct' => '\\O\\c\\t\\o\\b\\e\\r',
+		'Sep' => '\\S\\e\\p\\t\\e\\m\\b\\e\\r',
+		'apr' => 'apr',
+		'april' => 'Apr',
+		'aug' => 'aug',
+		'august' => 'Aug',
+		'before_yesterday' => 'Before yesterday',
+		'dec' => 'dec',
+		'december' => 'Dec',
+		'feb' => 'feb',
+		'february' => 'Feb',
+		'format_date' => '%s j\\<\\s\\u\\p\\>S\\<\\/\\s\\u\\p\\> Y',
+		'format_date_hour' => '%s j\\<\\s\\u\\p\\>S\\<\\/\\s\\u\\p\\> Y \\a\\t H\\:i',
+		'fri' => 'Fri',
+		'jan' => 'jan',
+		'january' => 'Jan',
+		'jul' => 'jul',
+		'july' => 'Jul',
+		'jun' => 'jun',
+		'june' => 'Jun',
+		'last_3_month' => 'Last three months',
+		'last_6_month' => 'Last six months',
+		'last_month' => 'Last month',
+		'last_week' => 'Last week',
+		'last_year' => 'Last year',
+		'mar' => 'mar',
+		'march' => 'Mar',
+		'may' => 'May',
+		'mon' => 'Mon',
+		'month' => 'months',
+		'nov' => 'nov',
+		'november' => 'Nov',
+		'oct' => 'oct',
+		'october' => 'Oct',
+		'sat' => 'Sat',
+		'sep' => 'sep',
+		'september' => 'Sep',
+		'sun' => 'Sun',
+		'thu' => 'Thu',
+		'today' => 'Today',
+		'tue' => 'Tue',
+		'wed' => 'Wed',
+		'yesterday' => 'Yesterday',
+	),
+	'freshrss' => array(
+		'_' => 'FreshRSS',
+		'about' => 'About FreshRSS',
+	),
+	'js' => array(
+		'category_empty' => 'Empty category',
+		'confirm_action' => 'Are you sure you want to perform this action? It cannot be cancelled!',
+		'confirm_action_feed_cat' => 'Are you sure you want to perform this action? You will lose related favorites and user queries. It cannot be cancelled!',
+		'feedback' => array(
+			'body_new_articles' => 'There are \\d new articles to read on FreshRSS.',
```

Comment:
The /dev branch is now using `%%d` instead of `\\d` https://github.com/FreshRSS/FreshRSS/blob/e3105f2135552701a4986873491a964b4e44c82e/app/i18n/en/gen.php#L111